tests/integration/golang/helpers/env.go

```diff
@@ -26,6 +26,14 @@ func GetS3EndpointUri() string {
 	return "http://localhost:9000"
 }
 
+func GetGSEndpointUri() string {
+	uri, ok := os.LookupEnv("FML_GS_ENDPOINT_URI")
+	if ok {
+		return uri
+	}
+	return "http://localhost:4333"
```

Comment:
I see that `FML_GS_ENDPOINT_URI` in the dev container points to `http://localhost:4443/storage/v1/` — which one is the good one?

Reply:
yep, this one is correct -> http://localhost:4443/storage/v1/. fixed.